thoth/common/openshift.py

```diff
@@ -1683,6 +1687,17 @@ def schedule_adviser(
             },
         )
 
+    def _assign_workflow_parameters(self) -> Dict[str, Any]:
```

Comment:
We should move scheduling of inspection workflow manager to this OpeNshift class. It logically should be placed here, this method would be also reused in that case.

Reply:
I can do that sure, I already open PR for adviser workflow as well, only thing we need to solve thoth-common software stack :(